scripts/stackstorm.js

```diff
@@ -180,7 +180,8 @@ module.exports = function(robot) {
         auth_client.on('expiry', authenticate);
       })
       .catch(function (err) {
-        err.message = 'Failed to authenticate: ' + err.message;
+        // Exit from invalid ST2_AUTH_USERNAME or ST2_AUTH_PASSWORD.
+        robot.logger.error('Failed credential authenticate.');
```

Comment:
"Failed credential authenticate" looks like not the best wording for error message.

Why `Failed to authenticate` message is not enough, just one line above?

Reply:
I was trying to tell end user what kind of authentication is failed (Not api key or token).

Reply:
`Failed credential authenticate.` message is confusing and not user-friendly at all.

If you want to mention explicitly initial auth method used, the following would work better:
```diff
- err.message = 'Failed to authenticate: ' + err.message;
+ err.message = 'Failed to authenticate with st2 username and password: ' + err.message;
```

However this is not accurate, because saying that we're not using token is not true.
I's something that's requested  via st2 username and password pair and really utilized in this specific block. Just look a few lines before: "Token received'.

Reply:
These code are only called with credential authentication, this is why I want to put more special message here to distinguish token and API key authentication. Will changed to `Failed to authenticate with st2 username and password: ' + err.message;`